src/main/target/FF_FORTINIF4/config.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#ifdef TARGET_CONFIG
+#include "telemetry/telemetry.h"
+
+void targetConfiguration(void)
+{
+    telemetryConfigMutable()->telemetry_inverted = false;
```

Comment:
Why are you doing this? `telemetry_inverted` defaults to false: https://github.com/betaflight/betaflight/blob/master/src/main/telemetry/telemetry.c#L60

This just adds unnecessary clutter to the target definitions and creates more work for developers doing target maintenance. Can you please remove this, or let me know if I should revert the pull request.